lib/grape/validations/validators/coerce.rb

```diff
@@ -56,7 +56,14 @@ def coerce_value(type, val)
         return val || Set.new if type == Set
         return val || {}      if type == Hash
 
-        converter = Virtus::Attribute.build(type)
+        # To support custom types that Virtus can't easily coerce, pass in an
+        # explicit coercer. Custom types must implement a `parse` class method.
+        converter_options = {}
+        if ParameterTypes.custom_type?(type)
+          converter_options[:coercer] = type.method(:parse)
+        end
```

Comment:
What are you referring to? It looks like [only a few cops are turned off](https://github.com/intridea/grape/blob/master/.rubocop_todo.yml) and I don't think any of them apply here.